src/main/java/com/github/caldav4j/CalDAVCollection.java

```diff
@@ -384,15 +380,11 @@ public String add(HttpClient httpClient, Calendar c, boolean attemptRetry)
 			throws CalDAV4JException {
 
 		Random random = new Random();
-		//
+		
 		// retry 3 times while caldav server returns PRECONDITION_FAILED
-		//
 		boolean didIt = false;
-		String path = "";
```

Comment:
If that's unused code, it's ok to remove it. Can you please create a new PR with unused-code removal, so we can address it easily?